package.json

```diff
@@ -16,7 +16,7 @@
   "license": "MIT",
   "devDependencies": {
     "standard": "^7.1.2",
-    "tap": "^12.0.1"
+    "tap": "12.1.1"
```

Comment:
Can you please restore the `^`?

Reply:
I remember this being something about Pino supporting node version >=4 and tap moved to >=8.

Is this something you can confirm?

Reply:
Confirmed, but `^` does not include semver-major changes.

Reply:
Reverted to `^12.0.1`.